tests/migrations/test_commands.py

```diff
@@ -334,10 +339,10 @@ def test_migrate_plan(self):
             'Planned operations:\n'
             'migrations.0003_third\n'
             '    Undo Create model Author\n'
-            '    Raw SQL operation -> SELECT * FROM migrations_book\n'
+            '    Raw SQL operation -> [\'SELECT * FROM migrations_book\']\n'
             'migrations.0002_second\n'
             '    Undo Create model Book\n'
-            '    Raw SQL operation -> SELECT * FROM migrations_salamander\n',
+            '    Raw SQL operation -> [\'SELECT * FROM migrations_salamand…\n',
```

Comment:
Could use double-quotes for these strings only to avoid the backslash while keeping the diff noise small.